### ISSUE TITLE
Add audio notes

### ISSUE THREAD

**Issue by trufae** [Antidote-for-Tox/Antidote#282]
When there's no text entered in the input box, the send button should be changed by a microphone and record small fragments of audio to be sent to the peer.

**Comment by dvor** [comment]
Maybe we want to add some special file type for that to the toxcore, e.g. now there are "data" and "avatar".

**Comment by trufae** [comment]
yep. and also be able to send videos, because right now it is only possible to send uncompressed images which is not really optimal
